src/Components/Artwork/__tests__/Details.test.tsx

```diff
@@ -101,20 +119,25 @@ const artworkInAuction = {
   partner: {
     name: "Forum Auctions",
     href: "/auction/forum-auctions",
-    __id: "UGFydG5lcjpmb3J1bS1hdWN0aW9ucw==",
   },
   sale: {
     is_auction: true,
-    is_live_open: false,
-    is_open: true,
     is_closed: false,
     display_timely_at: "ends in 14d",
-    __id: "U2FsZTpmb3J1bS1hdWN0aW9ucy1tdWx0aXBseQ==",
   },
   sale_artwork: {
-    highest_bid: { display: "$2,600", __id: null },
+    highest_bid: { display: "$2,600" },
     opening_bid: { display: "$2,400" },
-    __id: "U2FsZUFydHdvcms6Z2VyaGFyZC1yaWNodGVyLXR1bGlwcy1wMTctMTQ=",
+    counts: { bidder_positions: 0 },
   },
-  __id: "QXJ0d29yazpnZXJoYXJkLXJpY2h0ZXItdHVsaXBzLXAxNy0xNA==",
 }
```

Comment:
These `__id`s are probably be added by Relay at runtime, but aren't represented in the type system. Is this gonna break stuff ( /cc @ds300 ) ?

Reply:
They are added by Relay at compile (everything is done at compile time). _However_, unless you explicitly asked for them in your fragment, they won’t show up in the types emitted for your fragment. Nonetheless, they should be included.

Reply:
If you omit them, random uuids will be generated in their place. [Here, in the case of these tests](https://github.com/apollographql/graphql-tools/blob/b85137ba81c9bdc5667bced9cad9b1d226ea83fc/src/mock.ts#L57). Or [here, in the case of newer tests using the `mockData` option of `renderRelayTree`](https://github.com/artsy/reaction/blob/b7533dfcfe0b28df4ec8de0d472e5e0e4d44e0f5/src/DevTools/createMockNetworkLayer/index.ts#L94-L111)

That is normally fine, but might break tests which rely on disparate bits of mock data belonging to the same graphql node (they'll be stored in different places in the cache, potentially producing inconsistent views). I think as long as your tests still pass then deleting the `__id`s is OK, but might as well keep them just in case?